### ISSUE TITLE
google-it always returns empty array [] no matter what the query is

### ISSUE THREAD

**Issue by mattdellomo** [PatNeedham/google-it#16]
//this code it does not work at: https://npm.runkit.com/google

var googleIt = require("google-it");

googleIt({'query': 'coffee'}).then(results => {
  console.log(results);  //results(always) = "[]";
}).catch(e => {
 console.log(e); //no error is ever caught
});

**Comment by vyacheslav-lonschakov** [comment]
![response](https://user-images.githubusercontent.com/4928415/50225591-0b4a6e00-03b2-11e9-89fe-3e40c1a48dca.png)

**Comment by harshit28** [comment]
this issue is still there, anyone knows how to solve it?

**Comment by harshit28** [comment]
there is an issue in getResults() function 
just replace 
div.rc > h1.r > a with div.rc > div.r > a

**Comment by zalkanorr** [comment]
After the latest google update, here is my take on the code for it to work.

googleIt.js line 96-107

```
  // result titles
  var titles = $('div.rc > div.r > a');
  titles.each((index, elem) => {
    results.push({"title": $(elem).find('h3').text()})
  })

  // result links
  $('div.rc > div.r > a').map((index, elem) => {
    if (index < results.length) {
      results[index] = Object.assign(results[index], {"link": elem['attribs']['href']})
    }
  })
```

**Comment by kamilkoc-yusufayas** [comment]
any solotuion for this ? I'm getting stuck after some searches.

**Comment by yusuferenayas** [comment]
any solotuion for this ? I'm getting stuck after some searches.

**Comment by PatNeedham** [comment]
@yusuferenayas @mattdellomo @Jon-p1 this has now been fixed as of `google-it` v1.2.2 published on npm.

Thank you @zalkanorr for the title / link selector suggestions. In the subsequent version I'll make it dead simple to try out alternative selectors instead of the library being forced to use the hard-coded ones. For example via additional arguments:
```
$ google-it "olympics 2020" --titleSelector="div.rc > div.r > div.whateverNewThingComesNext > a"
```

**Comment by PatNeedham** [comment]
![image](https://user-images.githubusercontent.com/967811/71455612-c6df7400-2763-11ea-8915-a6ad73b23393.png)

`--titleSelector`, `--linkSelector`, and `--snippetSelector` options are now available as of v1.3.0
